src/background.js

```diff
@@ -34,12 +34,28 @@ async function disable(site) {
 let actions = {
     async message({domain, blocked}) {
         let site = await getSite(domain);
+        site.storage.blocked = blocked;
+        await browser.storage.sync.set({[site.domain]: site.storage});
         if (blocked) {
             await enable(site);
         } else {
             await disable(site);
         }
-        await browser.storage.sync.set({[site.domain]: {blocked}});
+    },
+
+    async navigation({url}) {
+        let site = await getSite(new URL(url).host);
+        if (site && site.blocked) {
+            let keys = Object.keys(site.storage);
+            let date = new Date().toISOString().substr(0, 16);
+            if (keys.length <= 5 && !keys.includes(date)) {
+                site.storage[date] = 1;
+                await browser.storage.sync.set({[site.domain]: site.storage});
+                if (keys.length >= 5 && !site.storage.blocked) {
+                    await disable(site);
+                }
```

Comment:
This section does not seem to disable the consent blocking (at least after a manual enable or messing around with the toggle)

I think it has something to do with the above if statement.

Reply:
Will be looking more into this in the next few hours

Reply:
Correct, this is also on purpose.  Manual action, either blocking or unblocking takes priority at all times, so we will not be _automatically_ allowing the site to show a consent dialog if the user has ever manually blocked it.